cadcdata/cadcdata/tests/test_core.py

```diff
@@ -142,22 +146,45 @@ def test_get_file(trans_reader_mock, basews_mock):
         assert expected_content == f.read()
     os.remove(file_name)
 
-    # test a get with decompress
+    # test a get with decompress and md5 check enabled
     file_name = 'bfile.txt'
-    file_content = 'ABCDEFGH12345'
-    file_chunks = [file_content[i:i + 5].encode()
-                   for i in xrange(0, len(file_content), 5)]
-    file_chunks.append('')  # last chunk is empty
+    file_content = 'aaaabbbb'
+    hash_md5 = hashlib.md5()
+    hash_md5.update(file_content.encode())
+    file_chunks = [file_content.encode(), ''.encode()]
+    decoded_file_content = 'MNOPRST6789'
+    decoded_file_chunks = [decoded_file_content.encode(), ''.encode()]
     response = Mock()
-    response.headers.get.return_value = 'filename={}.gz'.format(file_name)
+    response.headers = \
+        {'content-MD5': '{}'.format(hash_md5.hexdigest()),
```

Comment:
Just for clarification, the `Content-MD5` header is still supported within CADC?